bin/p2-schedule/main.go

```diff
@@ -53,12 +53,14 @@ func main() {
 			log.Fatalf("Could not read manifest at %s: %s\n", manifestPath, err)
 		}
 		path := kp.IntentPath(*nodeName, manifest.ID())
-		if *hookTypeName != "" {
+		if *hookTypeName == "global" {
```

Comment:
idk if kingpin lets you distinguish "specified flag without value (goes to default)" from "did not specify flag at all". I'll try it